src/components/BookImg/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+export default class BookImg extends Component {
+  render() {
+    if (this.props.src === null){
```

Comment:
`if (!this.props.src) {` works too if src is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)